app/assets/src/components/views/SampleUploadFlow/host_organism_message.scss

```diff
@@ -15,12 +17,25 @@
 .messageLine {
   margin: 0 $space-s;
   padding: $space-s;
-
-  &:nth-child(1) {
-    margin-top: $space-s;
-  }
+  font-size: $font-size;
+  margin: 0 44px; // align to right of icon
 
   &:nth-child(odd) {
     background-color: $info-dark;
   }
+
+  &:nth-child(even) {
+    background-color: $info-bg;
+  }
+
+  &:last-child {
+    margin-bottom: $space-m;
+  }
+}
+
+// NOTE: we should really have a Link component
+.link {
+  // Overwrite <a> inherit rule in header.scss
+  color: $primary-light !important;
```

Comment:
In any case color is a very specific attribute that is not that common throughout the page for links. Feel free to check with Jenn what is the most common value for color.